app/_hub/kong-inc/acl/_index.md

```diff
@@ -8,7 +8,7 @@ description: |
   [authentication plugin](/hub/#authentication) (such as
   [Basic Authentication](/hub/kong-inc/basic-auth/),
   [Key Authentication](/hub/kong-inc/key-auth/), [OAuth 2.0](/hub/kong-inc/oauth2/),
-  and [OpenID Connect](/hub/kong-inc/openid-connect/))
+  and [OpenID Connect](/hub/kong-inc/openid-connect/)/)
```

Comment:
looks like it doesn't handle the case when there is a ")" right at the end of the link very well :(